docs/zh/UserGuide/Operation Manual/DDL Data Definition Language.md

```diff
@@ -41,14 +41,15 @@ Msg: org.apache.iotdb.exception.MetadataErrorException: org.apache.iotdb.excepti
 
 ## 查看存储组
 
-在存储组创建后，我们可以使用[SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md)语句来查看所有的存储组，SQL语句如下所示：
+在存储组创建后，我们可以使用[SHOW STORAGE GROUP](../Operation%20Manual/SQL%20Reference.md)语句和[SHOW STORAGE GROUP <PrefixPath>](../Operation%20Manual/SQL%20Reference.md)来查看存储组，SQL语句如下所示：
```

Comment:
![image](https://user-images.githubusercontent.com/32640567/92699816-a2bed400-f380-11ea-8667-40875cc67244.png)
It seems that the `<PrefixPath>` doesn't display properly.

Reply:
Copy that ,i will improve it